packages/app/src/main/gameWatcher/refineLogFileData.ts

```diff
@@ -174,6 +174,9 @@ const processSoloLeaderboardStats = (
           copyLeaderboardStatsToLadderStats(leaderboardStat, soloData[k]);
           setMemberWithStatGroupMember(soloStatGroup.members[0], soloData[k].members[0]);
           matchingPlayerId = k;
+        } else if (soloData[k].members[0].relicID === soloStatGroup.members[0].profile_id) {
+          // stats for player not found, but we still want to populate some of his info
+          setMemberWithStatGroupMember(soloStatGroup.members[0], soloData[k].members[0]);
         }
       });
       unfinishedSoloDatas.delete(matchingPlayerId);
```

Comment:
I don't understand what is the point of this ? 😕

Reply:
This is just a partial fix. Since this requires the player to either have any solo game played ever or to be the first member of a team ranking by lucky chance. I will work on a proper fix